ReSharper.FSharp/src/FSharp.Psi.Features/src/Util/PsiUtil.fs

```diff
@@ -263,7 +270,7 @@ let isFollowedByEmptyLine (node: ITreeNode) =
 module PsiModificationUtil =
     /// Warning: newChild should not be child of oldChild
     let replace oldChild newChild =
-        ModificationUtil.ReplaceChild(oldChild, newChild) |> ignore
+        ModificationUtil.ReplaceChild(oldChild, newChild)
```

Comment:
The whole idea of this helper was not to require `|> ignore` in every call where we don't need the result node. Why not use the underlying method where you actually need the result?